### DISCUSSION TITLE
local development auth "Waiting for verification.."

### DISCUSSION THREAD

**Discussion by arowan** [supabase/supabase#5504] [Questions] [answered]
Hi all,

I've setup the CLI locally and have everything running DB wise. I'm just trying to get the auth to work, except I'm not receiving any emails. Are there some extra steps required to get it to work?

**Comment by delbertf** [comment, marked as answer]
Hi @arowan 

If you have activated the option to send confirmation mails, then you can open the Inbucket at http://localhost:54324 and click the confirmation link to activate the account or login. The local dev docker image will not send the mail to external mail servers (not sure if it can be configured to work). But with Inbucket you can catch the mails on your local host.

Tested with latest supabase 0.18.1

**Reply by geyermichael** [reply]
@delbertf I do not get a email verification on local development. Do you know how to activate it? Thought it is activated by default.

**Reply by ARMATAV** [reply]
Upgrade your CLI with `brew upgrade supabase` and you should get email verifications

**Reply by aguirrerjg** [reply]
It doesnt show email verifications after update

**Reply by o1Suleyman** [reply]
> It doesnt show email verifications after update

same problem
